cogboard-webapp/src/components/widgets/types/LogViewerWidget/index.js

```diff
@@ -23,9 +25,20 @@ const LogViewerWidget = ({ id }) => {
   const [searchFilter, setSearchFilter] = useState('');
   const [shouldFollowLogs, setFollow] = useState(true);
 
-  const logs = widgetData.content?.logs;
+  const storedLogs = useRef([]);
+
+  useEffect(() => {
+    getInitialLogs(id).then(logs => {
+      storedLogs.current = logs;
```

Comment:
dlaczego trzymamy logi w refie a nie stanie?? Wygląda to bardzo alarmująco

Reply:
Poprawiono, logi trzymane w stanie.